src/utils/exchange.js

```diff
@@ -77,3 +79,7 @@ export const parseOffer = (
     provider,
   };
 };
+
+export const isWethConvertedTx = (fromAssetSymbol: string, contractAddress: string): boolean => {
+  return fromAssetSymbol === 'ETH' && contractAddress === WETH[chainId].address;
```

Comment:
We could just do this on one line but it will likely bust the eslint `max-len` line limit. Just a note that this may be flagged up in future with the upcoming code standardisation work.